dtls_debug.h

```diff
@@ -138,8 +139,8 @@ void dtls_dsrv_log_addr(log_t level, const char *name, const session_t *addr);
 #define dtls_notice(...) LOG_INFO(__VA_ARGS__)
 #define dtls_info(...) LOG_INFO(__VA_ARGS__)
 #define dtls_debug(...) LOG_DEBUG(__VA_ARGS__)
-#define dtls_debug_hexdump(name, buf, length) dtls_dsrv_hexdump_log(DTLS_LOG_DEBUG, name, buf, length, 1)
-#define dtls_debug_dump(name, buf, length) dtls_dsrv_hexdump_log(DTLS_LOG_DEBUG, name, buf, length, 0)
+#define dtls_debug_hexdump(name, buf, length) { if (LOG_DEBUG <= LOG_LEVEL) { LOG_DEBUG("%s (%zu bytes):", name, length); od_hex_dump(buf, length, 0); }}
+#define dtls_debug_dump(name, buf, length) { if (LOG_DEBUG <= LOG_LEVEL) { LOG_DEBUG("%s (%zu bytes):", name, length); od_hex_dump(buf, length, 0); }}
```

Comment:
For tinydtls the difference of `dtls_debug_hexdump` and `dtls_debug_dump` is, that the first logs multiple lines (suitable for larger messages), while the second logs a single line. 
In zephyr I didn't found something to achieve that behavior.
For RIOT it may be possible, using "\n" at the end of the LOG_DEBUG and then use a proper `width` for each instead of the 0. If you don't feel well with that proposal, the current version will also do it for me.

Reply:
`od_hex_dump()` will already split the buffer into multiple lines on it's own. A width of 0 will just select the default with.

I added a newline to `dtls_debug_hexdump()`.